ts/modules/export-data.src.ts

```diff
@@ -471,6 +476,42 @@ Highcharts.Chart.prototype.getDataRows = function (
 
             return item.name + ((keyLength as any) > 1 ? ' (' + key + ')' : '');
         },
+        // Map the categories for value axes
+        getCategoryAndDatetimeMap = function (
+            series: Highcharts.Series,
+            pointArrayMap: Array<string>,
+            pIdx?: number
+        ): Highcharts.CategoryAndDatetimeMap {
+            var categoryMap: Highcharts.CategoryAndDatetimeMap['categoryMap'] =
+                {},
+                datetimeValueAxisMap:
+                Highcharts.CategoryAndDatetimeMap['datetimeValueAxisMap'] =
+                {};
```

Comment:
@bre1470 - can I do something better here? It's uber long, doesn't look good, hard to read etc.

Reply:
You can create some `type Alias = ...` before line 27 and use them in line 28 and 29 and here in line 485 and 488.
And regarding the naming I suggest to prefix all with `Exporting` because otherwise you will have a lot of fun to find it later on. For example the interface `CategoryAndDatetimeMap` should be better named `ExportingCategoryAndDatetimeMap`.

Reply:
I also think that the `And` in the name is not necessary.

Reply:
Maybe I am to much influenced by C#, but I find it strange to write `Datetime` instead of `DateTime`.

Reply:
Thanks!

Reply:
Changed.